### DISCUSSION TITLE
2022 Missing

### DISCUSSION THREAD

**Discussion by Lifinest7** [davetapley/helium-tax#85] [General]
Did u remove the 2022 export , its missing

**Comment by davetapley** [comment]
Yes, see https://github.com/davetapley/helium-tax/issues/83

**Comment by sisqonrw** [comment]
why does 2022 not work any more?

**Reply by davetapley** [reply]
I can no longer get prices reliably because the API is rate limited very aggressively.
As a temporary fix I did #83, but that means prices require manual updating, which I don't have time to work on.

**Comment by sisqonrw** [comment]
that was so fine before. 
i am not an expert to edit something.

https://davetapley.com/helium-tax/

that was so good for user like us.
what can i do?

**Comment by inventer2022** [comment]
hi it's very important tools for me.
is it possible to remove 2020 and add 2022?

**Comment by sisqonrw** [comment]
for me it is a important tool too

**Comment by shwang80** [comment]
Will this ever get fixed?

**Comment by sisqonrw** [comment]
try this

https://www.fairspot.host/hnt-export-mining-tax

can you tell me your hnt fees? i have 20 till 40. is it not to much?

**Reply by shwang80** [reply]
This is fine if you're only looking to get an aggregate total, but if you needed detailed rewards info for each miner, it doesn't work.

**Comment by sisqonrw** [comment]
My English is not that good. I'm from Germany.
Isn't every column correct?
Which column do I have to look at?

**Comment by sisqonrw** [comment]
how can i edit the tool or file?

**Comment by perezda** [comment]
Is there another site that will just export the mining income for a specific time range?

**Reply by shwang80** [reply]
yes. https://jstnryan.com/helium/
I think the API is still pretty restrictive, but I was able to collect 2 days at a time. My wallet has 10 active miners fyi. You may be able to export 20 days of data if you have 1 miner... just need to experiment.

**Comment by davetapley** [comment]
2022 is now added, let me know if anyone sees any problems.

**Reply by tsluder8** [reply]
Appreciate your work on this.  It seems to stall out around 4-5K transactions...

**Reply by davetapley** [reply]
@tsluder8 did you try a few times? I've found the API is aggressively rate limited, so sometimes it needs a few retries / patience.
